config/routes.rb

```diff
@@ -3,16 +3,30 @@
 
   get '/object/:id' => 'object#show', as: :object
 
-  constraints id: druid_regex, file_name: %r{[^/]+} do
-    get '/file/:id/:file_name' => 'file#show', as: :file
-    options '/file/:id/:file_name', to: 'file#options'
-    get '/file/app/:id/:file_name' => 'webauth#login_file'
-    get '/file/auth/:id/:file_name' => 'webauth#login_file', as: :auth_file
+  constraints id: druid_regex do
+    if Settings.features.allow_slashes_in_filenames
```

Comment:
why do we need this conditional? can we just set these routes as required?

Reply:
@jcoyne I'd be fine with that. Initially, one of the Access devs (maybe Chris?) asked if the feature could be behind a feature flag. LMK how you'd like to proceed.

Reply:
@cbeer why do you want a feature flag here?  Can't we just revert if it doesn't work?  Otherwise we need additional testing so that we hit both paths, right?

Reply:
@cbeer say something if you think this is required. Otherwise I think we should remove it to minimize complexity and avoid having to clean this up in the future.

Reply:
@jcoyne Removed in a follow-up commit.